include/boost/dynamic_bitset/dynamic_bitset.hpp

```diff
@@ -2106,8 +2119,26 @@ bool dynamic_bitset<Block, Allocator>::m_check_invariants() const
 
 } // namespace boost
 
-
 #undef BOOST_BITSET_CHAR
 
+// std::hash support
+#if (!defined(BOOST_NO_CXX11_HDR_FUNCTIONAL) || !defined(BOOST_NO_CXX11_NOEXCEPT)) && !defined(BOOST_DYNAMIC_BITSET_NO_STD_HASH)
```

Comment:
The "or" in this check is questionable. In any case the check for `BOOST_NO_CXX11_NOEXCEPT` doesn't seem worth it here but instead just use `BOOST_NOEXCEPT` instead of `noexcept` below.

Reply:
Thanks, fixed in the next push.